test/countable-set.test.ts

```diff
@@ -13,13 +13,7 @@ describe('CountableSet', () => {
       }
     `)
 
-    expect(s._map).toMatchInlineSnapshot(`
-      Map {
-        "bar1" => 1,
-        "bar2" => 2,
-        "bar3" => 1,
-      }
-    `)
+    expect(s._map).toMatchInlineSnapshot('Map {}')
```

Comment:
Why would this get affected?

Reply:
Just run `test:update`.

Reply:
What should i do for reset the test case? 😂

Reply:
It's funny, the test output may be related to the OS.
Before: inter i5
After: M1 pro